### DISCUSSION TITLE
PR reviewer is not working in yaml pipeline

### DISCUSSION THREAD

**Discussion by yehonatanblumenfeld** [mlarhrouch/azure-pipeline-gpt-pr-review#28] [Q&A]
I've installed the extension and tried to run the step in yaml pipeline, the step is running just fine and in the logs i could see that it added a few comments.
but when i go to the Pr page i cant see the comments there ,

but when i did the same process in the azure classic pipeline it worked perfectly, why is it ?

**Comment by mlarhrouch** [comment]
@yehonatanblumenfeld just tested the task in a yaml pipeline, the comments are added to the PR

**Reply by yehonatanblumenfeld** [reply]
So what am i doing wrong , i added the task to a yaml build and triggered it using PR, in the logs it writes the 'New comment added' but i cant see nothing in the PR

I've attached screen shot of the logs
![Screenshot 2023-08-08 at 16 55 20](https://github.com/mlarhrouch/azure-pipeline-gpt-pr-review/assets/76685805/18897a20-0fb9-415a-a99a-3c86b1d140c9)

**Comment by yehonatanblumenfeld** [comment]
Hey @mlarhrouch , it would be much appreciated if you can help me solve this, i've added the logs picture above.

**Reply by mlarhrouch** [reply]
@yehonatanblumenfeld i tested the task in yaml pipeline and the comments are added, i don't see anything wrong in the provided logs

**Reply by yehonatanblumenfeld** [reply]
Good morning @mlarhrouch , i've attached the screenshot so you can see that every thing works fine according to the logs, but it does not show them in the pr page. what could be the cause for that ?

screenshot of the pr page:

![Screenshot 2023-08-15 at 10 11 29](https://github.com/mlarhrouch/azure-pipeline-gpt-pr-review/assets/76685805/38f3b5fe-8a70-4c4b-b196-9534424a75ae)

**Reply by mlarhrouch** [reply]
did you add this to the yaml pipeline ?

<img width="1038" alt="image" src="https://github.com/mlarhrouch/azure-pipeline-gpt-pr-review/assets/12035600/dbb265a9-cfaa-47e0-8df0-1b151f8a8f9f">

and apply this : 

<img width="831" alt="image" src="https://github.com/mlarhrouch/azure-pipeline-gpt-pr-review/assets/12035600/cf2bc6e2-8d21-4bb1-9db1-bc597251e724">
